Go/README.md

```diff
@@ -1,74 +1,71 @@
 # Go
 
-[Go](https://github.com/golang/go) (also known as Golang) is an open source programming language maintained by Google.
+[Go](https://golang.org) is an open source programming language that makes it easy to build simple, reliable, and efficient software.
 
 ## Installation
 
-    $ brew install golang
+You can follow the [official instructions on how to install Go](https://golang.org/doc/install), or you can use homebrew instead:
+
+    brew install go
 
 When installed, try to run `go version` to see the installed version of Go.
 
+```sh
+$ go version
+go version go1.11.4 darwin/amd64
+```
+
 ## Setup your workspace
 
-### Add environment variables
+### The GOPATH and PATH environment variables
 
-Go has a unique approach of managing code where you have a single workspace for all your Go projects. For more information see the [documentation](https://golang.org/doc/code.html#Workspaces).
+The GOPATH environment variable specifies the location of your workspace. It defaults to a directory named `go` inside your home directory (`$HOME/go`).
 
-First, you'll need to tell Go the location of your workspace. We'll do this by adding some environment variables in your shell config file (usually `.bash_profile`, `.bashrc` or `.zshrc`).
+If you really want to change your GOPATH to something else add GOPATH to your shell/bash/zsh initialization file `.bash_profile`, `.bashrc` or `.zshrc`.
 
 ```sh
-export GOPATH=$HOME/go
-export GOROOT=/usr/local/opt/go/libexec
-export PATH=$PATH:$GOPATH/bin
-export PATH=$PATH:$GOROOT/bin
+export GOPATH=/something-else
 ```
 
-Once you save and close that file, make sure you run `source <filename>` to restart your bash session so that you have those changes. For example:
+Add `GOPATH/bin` directory to your `PATH` environment variable so you can run Go programs anywhere.
```

Comment:
> you can run Go programs **anywhere**.

Emphasis mine

I think *anywhere* needs to be replaced by something more meaningful.